src/GovUk.Education.ExploreEducationStatistics.Content.Api/Cache/GetThemesCacheKey.cs

```diff
@@ -0,0 +1,9 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Cache.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.Api.Requests;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Cache;
+
+public record GetThemesCacheKey : IMemoryCacheKey
+{
+    public string Key => $"{GetType().Name}";
```

Comment:
Not a biggy, but it's redundant to have this interpolated into a string. We're also using reflection unnecessarily as well:

Can just be:
```ts
public string Key => nameof(GetThemesCacheKey);
```

Reply:
Sorted ta

Reply:
Sorted ta